hazelcast/src/test/java/com/hazelcast/cache/CacheBasicAbstractTest.java

```diff
@@ -816,4 +817,22 @@ public void run() {
 
         abstract void doRun(Random random);
     }
+
+    // https://github.com/hazelcast/hazelcast/issues/7236
+    @Test
+    public void expiryTimeShouldNotBeChangedOnUpdateWhenCreatedExpiryPolicyIsUsed() {
+        final int CRETAED_EXPIRY_TIME_IN_MSEC = 1000;
```

Comment:
does it have to be 1000ms? could you make it 100ms or even less? 1s for single test is quite a lot.